spec.md

```diff
@@ -2540,19 +2538,20 @@ Interpolation can also be nested:
 Other extended shell-style features, such as `${VARIABLE/foo/bar}`, are not
 supported by the Compose specification.
 
-You can use a `$` (double-dollar sign) when your configuration needs a literal
-dollar sign. This also prevents Compose from interpolating a value, so a `$`
+You can use a `$$` (double-dollar sign) when your configuration needs a literal
```

Comment:
Oh this is an interesting (albeit unintentional 😛) fix

Reply:
CI fails because of this!